src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -496,6 +509,19 @@ public string Call(JObject opts, string? blockId)
             InvocationResult systemContractInvRes = _stateManager.SafeContext(() =>
             {
                 var snapshot = _stateManager.NewSnapshot();
+                if (!tx.Value.IsZero())
+                {
+                    var transferContext = new InvocationContext(source, snapshot, new TransactionReceipt
+                    {
+                        Block = snapshot.Blocks.GetTotalBlockHeight(),
+                        Transaction = tx
+                    });
+                
+                    var localInvocation = ContractEncoder.Encode("transfer(address,uint256)", destination, tx.Value);
+                    var transferResult =
+                        ContractInvoker.Invoke(ContractRegisterer.LatokenContract, transferContext, localInvocation, GasMetering.DefaultBlockGasLimit);
+                    gasUsed += transferResult.GasUsed;
```

Comment:
@sgladkov Same here
Should we rollback stateManager here using _stateManager.Rollback();?

Reply:
the same